### ISSUE TITLE
Got weird error message when connecting to xmage server launched by myself.

### ISSUE THREAD

**Issue by kkwezard** [magefree/mage#3540]
I launched a xmage server on my own VPS running centos 6.4.

I try connecting from my Windows PC but got a message below.

![qq 20170626115105](https://user-images.githubusercontent.com/2141813/27524340-be4ccf4e-5a66-11e7-9c4d-5a06b079b443.png)

Where this message is actually my linux machine's hostname.

I don't know what this message means.

**Comment by kkwezard** [comment]
Other details about this issue.
I launched the mage server with its default config, binding on interface 0.0.0.0:17171.
I also tried running this server on my Windows 10 PC machine and successfully connected it via local mage client and everything was OK.
However I still got this type of error message (display my machine's hostname) when trying to connect the server from another Windows PC which was not in the same LAN.
It seems that config only works for local loop or perhaps for machines in the same LAN(I didn't test this case).
